docker-compose.yml

```diff
@@ -18,8 +16,9 @@ services:
     image: postgres
     restart: always
     ports:
-      - 5432:5432
+      - "5432:5432"
     environment:
+      PORT: 5432
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: test
```

Comment:
If you are to use an insecure password, could you pretty please not expose the port to the outside world xD
I think you want to use the `expose` keyword instead of the `ports` one.

`ports` is binding your host machine's port to that container's port, whereas `expose` exposes a port of a container to other containers.

Reply:
this docker is only for local usage